static/CMakeLists.txt

```diff
@@ -6,16 +6,43 @@ set(pyArborX_STATIC_UTILITY_MODULES Experimental)
 
 # get source files for selected static modules
 foreach(PY_MODULE IN LISTS pyArborX_STATIC_GEOMETRY_MODULES pyArborX_STATIC_UTILITY_MODULES)
-  list(APPEND PY_SRC ${PROJECT_SOURCE_DIR}/src/pyArborX_${PY_MODULE}.cpp)
+  list(APPEND pyArborX_SRC ${PROJECT_SOURCE_DIR}/src/pyArborX_${PY_MODULE}.cpp)
   list(APPEND MODULE_INCLUDES "\n#include \"pyArborX_${PY_MODULE}.hpp\"" )
   list(APPEND MODULE_GENERATORS "\npyArborX::generate${PY_MODULE}Wrapper(m)")
 endforeach()
 
+#generate bindings for views of ArborX types
+set(LAYOUTS LayoutLeft LayoutRight)
```

Comment:
Why would we care to do both layout?

Reply:
These are rank-1 views

Reply:
reason was that I do not know when we will have an option in pykokkos-base to generate view-bindings for the ArborX types. If this extension to pykokkos-base takes longer we still can easily generate other view variants for our types.